app/controllers/runs_controller.rb

```diff
@@ -133,7 +133,7 @@ def set_run
                         {id: @run.user.id, name: @run.user.name}
                       end
 
-    gon.scale_to = @run.duration_ms(timing)
+    gon.scale_to = [@run.duration(timing).to_ms, @compare_runs.map { |run| run.duration(timing).to_ms }].flatten.max
```

Comment:
I'd prefer the splat operator over building an array that needs to be `flatten`ed:
```ruby
gon.scale_to = [@run.duration(timing).to_ms, *@compare_runs.map { |run| run.duration(timing).to_ms }].max
```

Reply:
Sounds good.